chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/listing/CertifiedProductDetailsEntity.java

```diff
@@ -259,582 +256,57 @@
-    }
-
-    public void setCertificationBodyName(final String certificationBodyName) {
-        this.certificationBodyName = certificationBodyName;
-    }
-
-    public Boolean getAcbIsRetired() {
-        return this.acbIsRetired;
-    }
-
-    public void setAcbIsRetired(final Boolean acbIsRetired) {
-        this.acbIsRetired = acbIsRetired;
-    }
-
-    public String getProductClassificationName() {
-        return productClassificationName;
-    }
-
-    public void setProductClassificationName(final String productClassificationName) {
-        this.productClassificationName = productClassificationName;
-    }
-
-    public String getPracticeTypeName() {
-        return practiceTypeName;
-    }
-
-    public void setPracticeTypeName(final String practiceTypeName) {
-        this.practiceTypeName = practiceTypeName;
-    }
-
-    public Long getProductVersionId() {
-        return productVersionId;
-    }
-
-    public void setProductVersionId(final Long productVersionId) {
-        this.productVersionId = productVersionId;
-    }
-
-    public String getProductVersion() {
-        return productVersion;
-    }
-
-    public void setProductVersion(final String productVersion) {
-        this.productVersion = productVersion;
-    }
-
-    public Long getProductId() {
-        return productId;
-    }
-
-    public void setProductId(final Long productId) {
-        this.productId = productId;
-    }
-
-    public String getProductName() {
-        return productName;
-    }
-
-    public void setProductName(final String productName) {
-        this.productName = productName;
-    }
-
-    public Long getDeveloperId() {
-        return developerId;
-    }
-
-    public void setDeveloperId(final Long developerId) {
-        this.developerId = developerId;
-    }
-
-    public String getDeveloperName() {
-        return developerName;
-    }
-
-    public void setDeveloperName(final String developerName) {
-        this.developerName = developerName;
-    }
+    @Column(name = "rwt_eligibility_year")
+    private Integer rwtEligibilityYear;
 
     public Date getCreationDate() {
         return Util.getNewDate(creationDate);
     }
 
-    public void setCreationDate(final Date creationDate) {
+    public void setCreationDate(Date creationDate) {
         this.creationDate = Util.getNewDate(creationDate);
     }
 
-    public Integer getCountCertifications() {
-        return countCertifications;
-    }
-
-    public void setCountCertifications(final Integer countCertifications) {
-        this.countCertifications = countCertifications;
-    }
-
-    public Integer getCountCqms() {
-        return countCqms;
-    }
-
-    public void setCountCqms(final Integer countCqms) {
-        this.countCqms = countCqms;
-    }
-
     public Date getLastModifiedDate() {
         return Util.getNewDate(lastModifiedDate);
     }
 
-    public void setLastModifiedDate(final Date lastModifiedDate) {
+    public void setLastModifiedDate(Date lastModifiedDate) {
```

Comment:
In other classes I've seen when we need it to use the non-Lombok-generated setter we annotate the field with 
`@Setter(AccessLevel.NONE)
 @Getter(AccessLevel.NONE)`